### ISSUE TITLE
Update link and revise installation Vagrantfile section of docs

### ISSUE THREAD

**Issue by willingc** [systers/vms#49]
http://vms.readthedocs.org/en/latest/Installation%20Guide/#download-systers-ubuntu-virtual-machine

This section of the doc should be updated to reflect current status. Also, all links should direct to a Systers/repo and should not direct to an individual's repo.

This issue is assigned to Vanda @veniemi1

**Comment by willingc** [comment]
@veniemi1 Here's the basic plan:

1. Follow the installation instructions to install vms and run tests as described in the Installation Guide.
2. Recommend edits and clarification to the installation guide.md file in the docs directory
3. Click Fork on the VMS repo to create a fork of the repo in your github account.
3. Create a new branch on your local system and edit the installation guide.md file
4. Commit and push your changes to your github account.
5. Submit a pull request to the systers/vms repo.

**Comment by veniemi1** [comment]
Thank you! This helps a lot.

**Comment by willingc** [comment]
@veniemi1 reports:
![screen shot 2015-06-16 at 5 35 17 pm](https://cloud.githubusercontent.com/assets/2680980/8197598/e5d3170e-144e-11e5-9e44-eeb84d37d441.png)
![screen shot 2015-06-16 at 5 34 56 pm](https://cloud.githubusercontent.com/assets/2680980/8197597/e5cf54e8-144e-11e5-81b2-7c4af6428541.png)

**Comment by willingc** [comment]
@veniemi1 Let's move the discussion from email to the issue here :)

Looking at the error message, it seems that vagrant does not believe that VirtualBox is installed or installed and configured correctly.

My recommendation would be to start at the beginning  of the installation guide http://vms.readthedocs.org/en/latest/Installation%20Guide/#installation-guide

Make sure that you have gone through each step. Or start over and go step by step. Leaving a message here with a screenshot if you have difficulty or errors with a step. Detailed instructions on installing VirtualBox are here: https://www.virtualbox.org/manual/ch02.html#installation_windows

**Comment by vubo** [comment]
@veniemi1 nice to meet you and work together on the VMS project :four_leaf_clover: 
Could you send me your readthedocs.org-username so I added you to the project.

**Comment by veniemi1** [comment]
I just created my account. My username is veniemi1, same as here.

**Comment by vubo** [comment]
@veniemi1 done! 

Now the Repository URL for the ReadTheDocs project is https://github.com/vubo/vms. This is my forked repository. To work with the docs, you should fork the develop branch from https://github.com/systers/vms and change the link in the project settings on the readthedocs.org. Your link will most likely look like this: https://github.com/veniemi1/vms

Good luck!

**Comment by willingc** [comment]
@vubo Thanks a bunch for helping out :cake: :smile: 

@veniemi1 Go Vanda go! I'm around today if you have any questions or wish to chat via Google Hangout.

**Comment by veniemi1** [comment]
The vagrant up command seems to be working through the Vagrantfile just fine until it gets to the config.vm.box = "systers-vms-dev" line. The exact error message is below. 
![vagrant_up_error](https://cloud.githubusercontent.com/assets/12129851/8255214/5822bd22-166c-11e5-8921-586d2491b90f.JPG)

**Comment by willingc** [comment]
@veniemi1 Let's check your network first...
Go to this link in the browser: http://54.183.32.240/vagrant/box/
You should see this output:
![screen shot 2015-06-19 at 7 33 20 am](https://cloud.githubusercontent.com/assets/2680980/8255585/c8539cfe-1655-11e5-8acc-04cd763de5be.png)

**Comment by willingc** [comment]
@veniemi1 If the prior step worked...

Go to the command line: enter ``vagrant destroy`` which will remove all cruft from the prior run of vagrant
After that completes: enter ``vagrant up``
Let me know the output.

**Comment by veniemi1** [comment]
When I went to the link you provided, everything was the same. However, the vagrant destroy and vagrant up yielded the same error message as earlier.

**Comment by willingc** [comment]
OK the good news it's not a networking issue.
I'm going to think about what would make sense to try next.

In the meantime, I want you to submit a pull request today to get used to the git workflow. Since the docs are not dependent on resolving the vagrant environment, please work on Issue #62. Please leave questions as they come up on Issue #62

**Comment by willingc** [comment]
@veniemi1 Another thing to try with Vagrant. Vagrant keeps all of its user data in a ``.vagrant.d`` directory http://docs.vagrantup.com/v2/installation/uninstallation.html. Please delete the ``.vagrant.d`` directory completely. 

Then try ``vagrant up``.

Just out of curiosity, what are you using for your terminal/command line? Powershell?

**Comment by willingc** [comment]
@veniemi1  I wrote a new Vagrantfile for you to try:
```
# -*- mode: ruby -*-
# vi: set ft=ruby :

Vagrant.configure("2") do |config|
  config.vm.box_url = "http://54.183.32.240/vagrant/box/systers-vms.box"
  config.vm.box = "systers-vms-dev"

  config.vm.provider "virtualbox" do |vb|
    vb.gui=true
  end

  config.vm.network "forwarded_port", guest: 80, host: 8080, auto_correct: true
  config.vm.network "forwarded_port", guest: 8000, host:8001
  config.vm.network "private_network", ip: "192.168.33.10"
  config.vm.network "public_network"
end
```

**Comment by veniemi1** [comment]
@willingc I'm having another issue with my vagrant. I got the vagrant up command to work and the VM is started... but it's asking me for a login and password even though I never created one when I installed either VirtualBox or Vagrant. This is preventing me from changing the directory. I've also tried moving on to the next step of installing the python-psycopg2 module. Is this another thing I need to download, or should the code in the instructions do the trick? Thank you for all your help.

**Comment by willingc** [comment]
@veniemi1 A few comments/questions...
1. Would you mind posting a screen shot of the errors or what is happening since that will help me :)
2. Which Vagrantfile are you using the one in the repo or the one above in the comments?
3. When you did ``vagrant up``, did you select option 1 for the networking question?
![screen shot 2015-06-21 at 10 46 31 pm](https://cloud.githubusercontent.com/assets/2680980/8276147/68792682-1867-11e5-8e2d-825379796f38.png)
4. Make sure you are changing directories in the correct window. You want to change directory ``cd /vagrant/vms`` (windows command may be slightly different)  in the same terminal window where you typed: ``vagrant up``, ``vagrant ssh``. See example here:
![screen shot 2015-06-21 at 10 50 34 pm](https://cloud.githubusercontent.com/assets/2680980/8276212/65fd4a40-1868-11e5-8a01-03b40c8f08a5.png)
Note: you do not type the change directory in the VirtualBox window.

As for Postgres and installing the python-psychopg2 module, you need to be able to have ``cd /vagrant/vms`` succeed before moving on to this.

Good luck. I'm traveling all day tomorrow so please leave questions and screenshots here in GitHub. Thanks!

**Comment by veniemi1** [comment]
I'm using the Vagrantfile that you posted. I got it to work up to the point of changing the directories but can't get that to work. I've searched on google for an equivalent windows command, but  haven't found anything useful. 
![directorychange](https://cloud.githubusercontent.com/assets/12129851/8285736/46690eec-18d3-11e5-8d30-42063032f15c.JPG)

**Comment by willingc** [comment]
@veniemi1 Well the good news is that you were successful loading the Vagrantfile and having it spin up a virtual machine. When working in the virtual machine, you should be able to just use Linux commands so it is the correct command ``cd /vagrant/vms``.

If you do an ``ls -al`` do you get the same printout that I got above.

Also, how are you getting a terminal window? Powershell? cmd?

**Comment by veniemi1** [comment]
I just did an ls -al and I got what you had above. I'm getting a terminal window through the command prompt. Also, quick side note, when I'm typing here on github how do I get the little boxes that specify that something is code rather than normal text?

**Comment by willingc** [comment]
@veniemi1 Now I am completely perplexed...
Can you type the following commands and there output (or screenshot):
1. ``pwd``
2. ``echo $PATH``
3. ``cd /``
4. ``ls``

To get something to come up as a code box you type `` at the start and `` at the end of the phrase. If you search GitHub Markdown, it will give you all the options :)

**Comment by veniemi1** [comment]
Alright, thank you! Here's what happened when I typed those commands in. I assumed you wanted them with vagrant still running. 
![test out commands](https://cloud.githubusercontent.com/assets/12129851/8291571/6afc2c2e-18f8-11e5-848e-1270d33d95bc.JPG)

**Comment by veniemi1** [comment]
@willingc I'm working on setting up PostgreSQL and I'm stuck on how to change the lines of code in the pg_hba.conf file. The code I am entering is `sudo nano /etc/postgresql/9.4.3/main/pg_hba.conf`. The numbers are the version of PostgreSQL that I have. When the file opens there is no text for me to change. Is this normal? Or, have I messed up somewhere?
![postgresql_screen](https://cloud.githubusercontent.com/assets/12129851/8319286/183a327c-19db-11e5-9cf8-cd2f7a5958f0.JPG)

**Comment by vubo** [comment]
@veniemi1 No, it should not be empty. This means that you enter the wrong path. Go to the /etc/postgresql:
```
cd etc
cd postgresql
```

and see the contents of this folder

```ls```

most likely, you have another version of the PostgreSQL and another folder.

**Comment by willingc** [comment]
@vubo Thanks for jumping in here. Much appreciated :)

@veniemi1 I'm at a conference this week so I only have wifi for a limited time in the evenings at my hotel. I will do my best to get back once a day.

I believe that @vubo is working on Windows so please follow her suggestions. Thanks :)

**Comment by veniemi1** [comment]
Okay, thank you. I'll take that into account as I'm working these next couple days. When I did the `ls` command  I got this below, but the file is still empty when I open it using 9.3 as the version number in the command. Do you have any suggestions?

![postgres](https://cloud.githubusercontent.com/assets/12129851/8333885/fc989b5c-1a62-11e5-8daa-ba7d533bf864.JPG)

**Comment by vubo** [comment]
@willingc yes, you're right, I am working on Windows and happy to help :) Nice conference :cherry_blossom:

@veniemi1 ok, this means that you need to enter
```
sudo nano /etc/postgresql/9.3/main/pg_hba.conf
```
Does It work?

**Comment by veniemi1** [comment]
It worked!

**Comment by veniemi1** [comment]
@willingc I've gotten through the rest of the postgresql setup. However, I'm having problems with the `cd /vagrant/vms` command again, which is annoying because I've gotten it to work before. I can get into the vagrant directory but not the vms directory. It says `No such file or directory`. For reference, I'm working on the Generate Database Tables section. Thank you for all your help. 
![capture](https://cloud.githubusercontent.com/assets/12129851/8342580/2e4757be-1a9b-11e5-8aaf-33edc61457aa.JPG)

**Comment by vubo** [comment]
@veniemi1 after:
```
cd vagrant
```
just do:
```
cd vms
```

**Comment by veniemi1** [comment]
My computer isn't finding the vms directory when I enter `cd vms`, and it doesn't appear to exist when I do the `ls -al` command. I also cannot find the manage.py file that is needed for the next line of code to work. 
![capture1](https://cloud.githubusercontent.com/assets/12129851/8345362/d07c792c-1abb-11e5-852d-d98137fc47d2.JPG)

**Comment by willingc** [comment]
@veniemi1 Just a thought...
Let's define for the moment 'Windows box' as your Windows computer and 'New box' for the virtual machine (where you see vagrant-ubuntu-trusty-32).

I am guessing that we have a very important step missing in the existing documentation (@vubo please confirm from your windows experience). 

When the ``vagrant up`` command is given on the 'Windows box', it should be given from the directory that holds the VMS files that were cloned from GitHub (delete the Vagrantfile and replace with the newer Vagrantfile that you have worked with this week).
![screen shot 2015-06-25 at 7 55 41 am](https://cloud.githubusercontent.com/assets/2680980/8357409/a2ad78f8-1b0f-11e5-8f52-57e04f555e64.png)
Basically when you type in ``vagrant up`` into your Windows terminal, the directory that you are in should have Vagrantfile as well as vms directory and docs directory.
![screen shot 2015-06-25 at 7 57 22 am](https://cloud.githubusercontent.com/assets/2680980/8357489/fb19728a-1b0f-11e5-92ba-96abdf2a0114.png)

I think this will solve the issue of not finding the ``vms`` directory when in the 'New box' virtual machine terminal.

Hang in there. This will be an important documentation improvement if I am correct ;)

**Comment by vubo** [comment]
@willingc @veniemi1 yes, I start vagrant from the "vms" and "vagrantfile" directory. Here ist it (V:\DJANGO\vms):
![vagrant](https://cloud.githubusercontent.com/assets/7673323/8359947/de280808-1b69-11e5-9b36-1e8bffa0aff6.png)

And I do:
![vag2](https://cloud.githubusercontent.com/assets/7673323/8360064/9ea3e62e-1b6a-11e5-98f8-bd15b1a7f7ac.png)
![vag3](https://cloud.githubusercontent.com/assets/7673323/8360070/a24e89a0-1b6a-11e5-8d02-998643d803e7.png)

I hope, it helps!

**Comment by veniemi1** [comment]
@willingc I fixed the issue with cloning the repository, but now can't get the line `python manage.py syncdb` to work because it's saying that password authentication failed. I also can't get my computer to recognize the `\dt` command inside the postgres or `python manage.py sqlall app_name_here` command. On the last command, what app name should I use?
![capture3](https://cloud.githubusercontent.com/assets/12129851/8368379/6c6df64c-1b77-11e5-9ed6-6e6775ee45e0.JPG)
`\dt` command
![capture2](https://cloud.githubusercontent.com/assets/12129851/8368380/76bae2cc-1b77-11e5-9c50-09f89489bcf6.JPG)
`python manage.py syncdb` command

**Comment by vubo** [comment]
@veniemi1 in the file /vms/vms/settings.py:
´´´
# Database
# Change these database settings if your database engine, database name,
# username or password changes
DATABASES = {
    'default': {
        'ENGINE': 'django.db.backends.postgresql_psycopg2',
        'NAME': 'vms',
        'USER': 'myuser',
        'PASSWORD': 'mypassword',
        'HOST': 'localhost',
    }
}
´´´
you need to change "myuser" and "mypassword" to your admin login and password.

**Comment by veniemi1** [comment]
I'm done! I got everything in the instructions to work.

**Comment by smarshy** [comment]
@tapasweni-pathak I think the suggested changes in docs here and in #72 are related and have been merged.
